src/geometry/geom2/create.test.js

```diff
@@ -0,0 +1,16 @@
+const test = require('ava')
+
+const {create} = require('./index')
+
+test('create: Creates an empty, uncanonicalized geom2', t => {
+  const expected = {baseSides: [], sides: [],
+                    isCanonicalized: false, transforms: new Float32Array([1,0,0,0,0,1,0,0,0,0,1,0,0,0,0,1]) }
```

Comment:
you can use a mat4.create instead of the manual floatArray right ? or did you want it explicit ?